src/appengine/libs/helpers.py

```diff
@@ -164,6 +164,15 @@ def get_integer_key(request):
     raise EarlyExitException('Invalid key format.', 400)
 
 
+def get_integer_key_flask(request):
+  """Convenience function for getting an integer datastore key ID."""
+  key = request.form.get('key')
```

Comment:
In this case it makes more sense to do a type check on `request` to decide the behaviour rather than breaking things out into a completely different one. Just add a TODO(username) comment to remember to clean up this up.

Reply:
done